infra/token-transfer-server/src/tasks/transfer.js

```diff
@@ -0,0 +1,82 @@
+const moment = require('moment')
+
+const {
+  largeTransferThreshold,
+  largeTransferDelayMinutes
+} = require('../config')
+const { Transfer, TransferTask, Sequelize } = require('../models')
+const { executeTransfer } = require('../lib/transfer')
+const logger = require('../logger')
+const enums = require('../enums')
+
+const executeTransfers = async () => {
+  logger.info('Running execute transfers job...')
+
+  const outstandingTasks = await TransferTask.findAll({
+    where: {
+      end: null
+    }
+  })
+  if (outstandingTasks.length > 0) {
+    throw new Error(
+      `Found incomplete transfer task(s), wait for completion or clean up manually.`
+    )
+  }
+
+  const transferTask = await TransferTask.create({
```

Comment:
should this row insertion get moved to line 47, after the test on no unconfirmed transfer?

Reply:
Yep my bad 👍